_opcodes/op195-bgee.html

```diff
@@ -0,0 +1,37 @@
+---
+n: 195
+name: "Familiar Bond"
+param1: "Max HP Modifier"
+param2: "Unused"
+param3: "Master ID (owner)"
+bg1: 0
+bg2: 0
+bgee: 1
+iwd1: 0
+iwd2: 0
+pst: 0
+---
+When this creature dies, the following is applied to <code>Master ID</code>:
+<ul>
+	<li>Floats Text String <a href="../files/2da/2da_bgee/enginest.htm#990">STRREF_GUI_MIXED_FAMILIARDIED</a>.</li>
+	<li><a href="#op10">Opcode #10</a>: <code>Statistic Modifier = -1</code>, <code>Type = 0</code>, <code>Timing Mode 1</code>.</li>
+	<li><a href="#op18">Opcode #18</a>: <code>Statistic Modifier = Max HP Modifier</code>, <code>Type = 0</code>, <code>Timing Mode 1</code>.</li>
+	<li><a href="#op12">Opcode #12</a>: <code>Damage Amount = 'this creature's Max HP'</code>, <code>Type = MAGIC</code>, <code>Timing Mode 1</code>, <code>Save Type = Bypass Mirror Image</code>.</li>
+</ul>
+
+Known values for <code>Master ID</code> are:
+<ul>
+	<li>-1 &longrightarrow; Not Party Member</li>
+	<li>0 &longrightarrow; Player1</li>
+	<li>1 &longrightarrow; Player2</li>
+	<li>2 &longrightarrow; Player3</li>
+	<li>3 &longrightarrow; Player4</li>
+	<li>4 &longrightarrow; Player5</li>
+	<li>5 &longrightarrow; Player6</li>
+</ul>
+
+{% capture note %}
+<code>Parameter2</code> is possibly intended to be <code>Damage Type</code>. It uses the same format as <a href="#op12">opcode #12</a> (<a href="#op192">opcode #192</a> sets it to <code>0x400000 – MAGIC</code>), but changing it has no effect.
+{% endcapture %}
+
+{% include note.html %}
```

Comment:
This can replace the main 195; I don't see anything new.

Reply:
Done.